CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+### 0.15.2 (2021-05-12)
+* Add `offline_since` to address_metadata endpoint if node is offline. Returns time in seconds.
```

Comment:
I think the [actual key](https://github.com/raiden-network/raiden-services/blob/master/src/pathfinding_service/api.py#L476) is `seen_offline_since`